tests/data/make_butler_repo.py

```diff
@@ -0,0 +1,29 @@
+#!/bin/env python
+# Make a gen3 butler and import the files to have a working cfht_minimal repo.
+#
+# This can be used to regenerate the gen3 registry when the butler has changes
+# that are not backwards compatible.
+#
+# I had to temporarily alter the sdss-dr9-fink-v5b.ecsv file to have absolute
+# paths, because I couldn't get ingest_files with `prefix` no matter what I did
+# on this script, even though it works for the tests.
+
+import lsst.daf.butler
+from lsst.daf.butler.script import ingest_files
+import lsst.obs.base
+
+repopath = "cfht_minimal/repo"
+instrument = "lsst.obs.cfht.MegaPrime"
+lsst.daf.butler.Butler.makeRepo(repopath)
+butler = lsst.daf.butler.Butler(repopath, writeable=True)
+instrInstance = lsst.obs.base.utils.getInstrument(instrument)
+instrInstance.register(butler.registry)
+
+graph = butler.registry.dimensions.extract(["htm7"])
+datasetType = lsst.daf.butler.DatasetType("sdss_dr9_fink_v5b", graph,
```

Comment:
Seeing how often `sdss_dr9_fink_v5b` appears everywhere, would it make sense to define a `refcat_default = "sdss_dr9_fink_v5b"` somewhere in the package?

Reply:
In the package, or just in this file? The sdss refcats are really only used in a couple of tests, so they definitely shouldn't be package-wide defaults. Those are Gaia DR2 for astrometry and PS1 for photometry.

Reply:
Oh, fair enough. Maybe we should have some refcat names defined in a package somewhere because it seems like it'll be a huge pain if we ever want to update them en masse.

Reply:
Eh, these particular refcats will probably never get updated, since they are each for a specific testing purpose. If we add e.g. a catalog with proper motion covariances or something, I'll just add a specific test for that catalog.

For non-jointcal packages, I think we'll be consolidating some of those names as part of the drp_pipe work, and I plan to tackle DM-27013 over the break, which should help.